Lib/fontTools/varLib/__init__.py

```diff
@@ -763,19 +824,22 @@ def load_designspace(designspace):
 	normalized_master_locs = [models.normalizeLocation(m, internal_axis_supports) for m in internal_master_locs]
 	log.info("Normalized master locations:\n%s", pformat(normalized_master_locs))
 
-	# Find base master
-	base_idx = None
+	# Find default master after normalization.
+	# TODO: Can the base master ever be anything other than what ds.findDefault() returns?
```

Comment:
I remember a discussion about this and also vs. copyKerning = true and friends I think

Reply:
Cosimo says this code may predate findDefault and what they do should match. Should add a test for that.